hive-tables-creation.py

```diff
@@ -0,0 +1,44 @@
+import boto3
```

Comment:
Do we want to create these in Python, or perhaps manage these using Terraform? https://www.terraform.io/docs/providers/aws/r/glue_catalog_table.html can probably do the job. I honestly don't know whether this is infra-code or app-code so can be persuaded either way.

Reply:
Going forward we will have to create  hive tables dynamically for all hbase tables . With Python we can fetch all hbase tables  using  module happy base and create corresponding hive tables